web-server/pages/api/resources/orgs/[org_id]/teams/v2.ts

```diff
@@ -30,8 +40,15 @@ const postSchema = yup.object().shape({
           yup.array().of(
             yup.object().shape({
               idempotency_key: yup.string().required(),
+              deployment_type: yup.string().required(),
               slug: yup.string().required(),
-              name: yup.string().required()
+              name: yup.string().required(),
+              repo_workflows: yup.array().of(
+                yup.object().shape({
+                  name: yup.string().required(),
+                  value: yup.string().required()
+                })
+              )
```

Comment:
This and the below schema looks identical. Maybe we can assign it to a variable and use it from there?

Reply:
moved to a var